service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/azure/container/CopyAzureStorageContainerDefinitionStep.java

```diff
@@ -127,14 +150,6 @@ public StepResult doStep(FlightContext flightContext)
     return StepResult.getStepResultSuccess();
   }
 
-  private void validateInputs(FlightMap inputParameters, FlightMap workingMap) {
-    FlightUtils.validateRequiredEntries(
```

Comment:
It's my understanding that we want to call `validateRequiredEntries` at the top of every step -- is that not the case?

Reply:
i replaced that with getRequired which throws too.